Update incident commander assignment copy (#24368)
DataDog/documentation: content/en/service_management/incident_management/_index.md

In the creation modal, you add a descriptive title, select whether customers were impacted (yes, no, or unknown) and select a severity level (1-5, unknown).

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander (IC). If the person declaring an incident is not a member of a Datadog account, then the IC becomes unassigned. The IC can be changed on the [incidents page][1] later if necessary.

Reviewer: ```suggestion
If the user declaring the incident has connected their Slack to their Datadog account, then by default that user becomes the Incident Commander (IC). If the person declaring an incident is not a member of a Datadog account, then the IC is unassigned. You can change the IC on the [incidents page][1] later if necessary.
```

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed later in-app if necessary.

Author: ```suggestion
If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed on the [incidents page][1] later if necessary.
```

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed on the [incidents page][1] later if necessary.

Reviewer 2: i don't think it makes sense to call it "anonymous Slack declaration" here since the setting is called 
"Allow users to declare incidents without a connected Slack account". i think we could leave this out of this section maybe since the incident can't be declared if this setting is off(is that right?)

Author: The second point is true. I think the main issue is that the setting is never mentioned anywhere so it reads to me that the incident _is_ declared regardless of that setting.

Reviewer 2: okay fair. maybe the settings part should go [here](https://docs.datadoghq.com/service_management/incident_management/incident_settings#slack-features) though? like this it reads kind of like the IC being unassigned has something to do with that setting, not the actual incident getting declared

Author: Agree. I'll go ahead and leave it out of this copy though and won't mention it in my original copy to prevent an endless feedback loop as most of this documentation is already stale.

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed on the [incidents page][1] later if necessary.

Author: ```suggestion
If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account, then the IC becomes unassigned. The Incident Commander (IC) can be changed on the [incidents page][1] later if necessary.
```

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned and can be assigned to another IC in-app.

Author: ```suggestion
If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed later in-app if necessary.
```

> If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account _and_ anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned and can be assigned to another IC in-app.

Author: ```suggestion
If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned and can be assigned to another IC in-app.
```

> Once you declare an incident from Slack, it generates an incident channel.

Author: ```suggestion
After you declare an incident from Slack, it generates an incident channel.
```
